packages/icons/package.json

```diff
@@ -4,13 +4,28 @@
    "main": "./index.tsx",
    "types": "./index.tsx",
    "license": "MIT",
+   "scripts": {
+      "clean": "",
+      "build": ""
+   },
    "peerDependencies": {
-      "@chakra-ui/react": ">= 1.8.3 < 2"
+      "@chakra-ui/react": ">= 1.8.3 < 2",
+      "@emotion/react": ">=11.0.0 <12.0.0",
+      "framer-motion": ">=3.0.0 <4.0.0 || >=4.0.0 <5.0.0 || >=5.0.0 <6.0.0 || >=6.0.0 <7.0.0",
```

Comment:
Would it have worked to say `>=3.0.0 <7.0.0`?  Also, why do we need these to be peerDeps and devDeps?

Reply:
You're right, I didn't catch that. Fixed!

To answer your second question, I'll point you to this comment from @andyg0808 : https://github.com/iFixit/react-commerce/pull/633#discussion_r950431392